database/migrations/2023_06_01_234455_create_meetings_table.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateMeetingsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('meetings', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->unsignedBigInteger('room_id');
+            $table->unsignedBigInteger('category_id');
+            $table->unsignedBigInteger('level_id');
+            $table->string('title');
+            $table->timestamp('date');
+            $table->unsignedBigInteger('start_at');
+            $table->integer('status_id')->default(1); //1:stand by
```

Comment:
what is the possible values for `status_id`?

Reply:
start_at - allocate and arrange numbers within a designated time slot, such as an hour on a schedule.
status_id - in session, stand by, done